meta-luneos/recipes-webos/chromium/files/0003-run_app_shell-fix-XDG_RUNTIME_DIR-for-LuneOS.patch

```diff
@@ -33,5 +33,5 @@ index 7ac65702b1..b34863bb35 100644
  
  DEPLOY_DIR=$(dirname $(readlink -f $0))
 -- 
-2.17.1
+2.37.3.windows.1
```

Comment:
Please use git format-patch --no-numbered --no-signature

Reply:
Done ;) But your 2 patches do the same :P